### DISCUSSION TITLE
Detach and move chat box

### DISCUSSION THREAD

**Discussion by BuzzfeedBlue** [runelite/runelite#14908] [Q&A] [answered]
Id like to have the option to move/pop out the chat box and drag/dock like any other element in game. I dont know the back end required to move the original runescape element of the chat box, but im sure you could grab the data and display it in an entirely new runelite chatbox if it were an issue.

**Comment by mattthw** [comment, marked as answer]
I am able to do this by holding ALT + drag

![image](https://user-images.githubusercontent.com/5104620/182244182-b9381e20-b688-4f3e-80ed-0b37408d9c5f.png)

**Reply by BuzzfeedBlue** [reply]
yeah, since i opened this thread, they added the feature

**Reply by Nightfirecat** [reply]
Please note this is not (and will likely never be) available for the fixed viewport.
